### PR TITLE
Sync Serverless CI config/

### PR DESCRIPTION
**Pull request by serverless-qe**

### CODE REVIEW

**Review by creydr — APPROVED** [review]
/lgtm

File: ci-operator/config/openshift-knative/kn-plugin-func/openshift-knative-kn-plugin-func-release-next__415.yaml [hunk 1]
```diff
@@ -34,7 +64,7 @@ tests:
         openshift/e2e_oncluster_tests.sh
       dependencies:
       - env: FUNC_UTILS_IMG
-        name: knative-kn-plugin-func-util
+        name: knative-kn-plugin-func-func-util
```
[the remark this answers is not in the document]
Comment (pierDipi) [reply ci-operator/config/openshift-knative/kn-plugin-func/openshift-knative-kn-plugin-func-release-next__415.yaml:67]: naming scheme is `<prefix><repo>-<image-name>`

**Pull request merged**